deepinv/models/bm3d.py

```diff
@@ -5,8 +5,8 @@
 # Compat for optional dependency on BM3D
 try:
     import bm3d
-except ImportError as e:
-    bm3d = e
+except:
+    bm3d = ImportError("The bm3d package is not installed.")
```

Comment:
So `except ImportError as e:` is unstable but this way it is stable?

Reply:
it is to avoid the case where it fails during the import, but raises an exception that is not necessary an ImportError (it was actually happening to me)